src/modules/infobox/components/FeatureElevationProfileInformation.vue

```diff
@@ -72,14 +72,17 @@ export default {
                     {
                         title: 'profile_distance',
                         icons: [
-                            ['far', 'eye'],
```

Comment:
The icon `eye` should also be removed from the `setup-fontawesome` list if not used anymore